microsetta_public_api/models/tests/test_taxonomy.py

```diff
@@ -277,6 +277,33 @@ def test_get_group_missing(self):
         with self.assertRaisesRegex(UnknownID, "sample-X does not exist"):
             taxonomy.get_group(['sample-X'])
 
+    def test_get_counts(self):
+        taxonomy_df = pd.DataFrame([['feature-1', 'k__a; p__b; c__c', 0.123],
+                                    ['feature-2',
+                                     'k__a; p__b; c__c; o__d; f__e', 0.345],
+                                    ['feature-3', 'k__a; p__f; c__g; o__h',
```

Comment:
Can you add/modify a feature here that has a weird taxonomy label? Like `'p__b; c__c; o__d'`?

Reply:
That doesn't make sense

Reply:
The number of ways taxonomy can be messed up is infinite. We only support well structured taxonomies

Reply:
Okay.